Clients/GUI/Scripts/SignUpper.cs

```diff
@@ -1,13 +1,18 @@
 using DTO;
 using HttpWebRequestWrapperLib;
+using System;
+using System.Runtime.InteropServices;
+using System.Text.Json;
 
 namespace GUI.Scripts
 {
     public static class SignUpper
     {
-        public static void SignUp(UserEmailPassword data)
+        public static Guid SignUp(UserEmailPassword data)
         {
-            new HttpWebRequestWrapper().Post("https://localhost:5011/user/create", null, data);
+            var wrapper = new HttpWebRequestWrapper();
+            var response = wrapper.Post("https://localhost:5011/user/create", null, data);
+            return JsonSerializer.Deserialize<Guid>(response);
```

Comment:
I am not sure, but here we don't need try catch?

Reply:
> I am not sure, but here we don't need try catch?

We try catch in the page's code